### PR TITLE
GEOMESA-1514 Standardizing tools output/logging

### PR DESCRIPTION
**Pull request by elahrvivaz**
~~Note: stacks on top of #1222~~

Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-accumulo/geomesa-accumulo-tools/conf/log4j.properties [hunk 1]
```diff
@@ -1,33 +0,0 @@
-# log to stderr to be consistent with unix (data to stdout + logging to stderr)
-log4j.rootLogger=info,file
-
-geomesa.log.dir=.
-
-# set logging levels to appropriate values
-log4j.logger.org.locationtech.geomesa.tools.accumulo.ingest=info,file
-log4j.additivity.org.locationtech.geomesa.tools.accumulo.ingest=false
```
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-tools/conf/log4j.properties]: The root logger is configured to just print to the log file - there is a 'user' logger that should be used for things the user will see (usually validation errors, info messages, etc), and there is an 'output' logger for results. All other loggers will go to the logs.

**Comment by elahrvivaz** [comment]
rebased on master

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/stats/StatsHistogramCommand.scala [hunk 2]
```diff
@@ -276,7 +272,7 @@ object StatsHistogramCommand {
     """                         ,,,________,.:__.,.,,:: ,                 .:,________________________________,,,__________,,                       """,
     """                          __________.,.: .,__,/:                    __.,,,____,:;__.:,.,;__.,:,__________________,, :                       """,
     """                          .,__________,__.,                      ,,.:  : , .::;.,:.,__:,.,__________________,:__   __                       """,
-    """                           ,println(,,__________.,,                       :,,__.      :,;__________________________,,  ,.:,,                        """,
```
Comment (jnh5y) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/stats/StatsHistogramCommand.scala]: Is this intentional?

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/DescribeSchemaCommand.scala [hunk 3]
```diff
@@ -22,22 +22,16 @@ trait DescribeSchemaCommand[DS <: DataStore] extends DataStoreCommand[DS] {
...
-  override def execute(): Unit = {
-    logger.info(s"Describing attributes of feature '${params.featureName}'")
-    try {
-      withDataStore(describe)
-    } catch {
-      case p: ParameterException => throw p
-      case NonFatal(e) => logger.error(s"Couldn't describe feature '${params.featureName}':", e)
-    }
-  }
+  override def execute(): Unit = withDataStore(describe)
```
Comment (jnh5y) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/DescribeSchemaCommand.scala]: With this refactoring pattern, are log messages preserved?

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/stats/StatsHistogramCommand.scala (hunk 2 above)
Comment (elahrvivaz) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/stats/StatsHistogramCommand.scala]: yeah, that 'println' got inserted at some point and we didn't notice... probably a find/replace gone awry

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/DescribeSchemaCommand.scala (hunk 3 above)
Comment (elahrvivaz) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/status/DescribeSchemaCommand.scala]: yeah, they are caught at the Runner level now

**Pull request merged**